design_doc.go

```diff
@@ -19,6 +19,7 @@ type ViewMap map[string]ViewDef
 type DesignDocOptions struct {
 	LocalSeq      bool `json:"local_seq,omitempty"`
 	IncludeDesign bool `json:"include_design,omitempty"`
+	Raw           bool `json:"raw,omitempty"`
```

Comment:
Do I need a parallel implementation of this in https://github.com/couchbaselabs/walrus/blob/master/design_doc.go#L19